renovate requires node.js 20.15.1 or higher

renovate requires node.js 20.15.1 or higher, but node.js installed by default on ubuntu-latest is 18.20.4, so renovate exits with an error.

If we have an older version of nodejs, we will need to install the appropriate nodejs.

```
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'renovate@38.14.0',
npm warn EBADENGINE   required: { node: '^20.15.1', pnpm: '^9.0.0' },
npm warn EBADENGINE   current: { node: 'v18.20.4', npm: '10.7.0' }
npm warn EBADENGINE }
```

**Detailed logs**

```
Run npx renovate \
  npx renovate \
    --schedule= \
    --require-config=ignored \
    --dry-run korosuke613/homepage-2nd \
    | tee /home/runner/work/_temp/renovate-dry-run.log
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    RENOVATE_USERNAME: GitHub Action
    RENOVATE_GIT_AUTHOR: GitHub Action <action@github.com>
    RENOVATE_TOKEN: ***
    LOG_LEVEL: debug
    LOG_FORMAT: text
    RENOVATE_CONFIG_FILE: renovate.json5
npm warn exec The following package was not found and will be installed: renovate@38.14.0
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'renovate@38.14.0',
npm warn EBADENGINE   required: { node: '^20.15.1', pnpm: '^9.0.0' },
npm warn EBADENGINE   current: { node: 'v18.20.4', npm: '10.7.0' }
npm warn EBADENGINE }
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated rimraf@2.4.5: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@6.0.4: Glob versions prior to v9 are no longer supported
DEBUG: Using RE2 regex engine

...

 INFO: Renovate is exiting with a non-zero code due to the following logged errors
       "loggerErrors": [
         {
           "name": "renovate",
           "level": 50,
           "logContext": "Q5DmnlZ4u7Jz3ss-Ydw5-",
           "versions": {
             "node": "18.20.4",
             "acorn": "8.11.3",
             "ada": "2.7.8",
             "ares": "1.28.1",
             "base64": "0.5.2",
             "brotli": "1.0.9",
             "cjs_module_lexer": "1.2.2",
             "cldr": "44.1",
             "icu": "74.2",
             "llhttp": "6.1.1",
             "modules": "108",
             "napi": "9",
             "nghttp2": "1.61.0",
             "nghttp3": "0.7.0",
             "ngtcp2": "1.3.0",
             "openssl": "3.0.13+quic",
             "simdutf": "5.2.4",
             "tz": "2024a",
             "undici": "5.28.4",
             "unicode": "15.1",
             "uv": "1.44.2",
             "uvwasi": "0.0.19",
             "v8": "10.2.154.26-node.37",
             "zlib": "1.3.0.1-motley"
           },
           "range": "^20.15.1",
           "msg": "Unsupported node environment detected. Please update your node version."
         }
       ]
Error: Process completed with exit code 1.
```

### Version
- Workflow version: 'cybozu/renovate-dry-run-action@v2' (SHA:`375d1355584ea033daac09cbbeecf3ffe772d8a3`)
- Runner: `ubuntu-latest`

--- a/action.yaml
+++ b/action.yaml
@@ -37,6 +37,10 @@ inputs:
     description: 'Renovate git author'
     required: false
     default: "GitHub Action <action@github.com>"
+  install-lts-nodejs:
+    description: 'if `true`, install LTS Node.js. Because Renovate may require more than the default Node.js version of GitHub Actions.'
+    required: false
+    default: "true"
 
 runs:
   using: "composite"
@@ -53,6 +57,15 @@ runs:
         
         # Create a path to save a action summary.
         echo "summary=${{ runner.temp }}/summary.md" >> $GITHUB_OUTPUT
+
+    # Setup LTS Node.js
+    #   Because Renovate may require more than the default Node.js version of GitHub Actions.
+    #   see: https://github.com/cybozu/renovate-dry-run-action/issues/26
+    - name: Setup LTS Node.js
+      if: inputs.install-lts-nodejs == 'true'
+      uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
 
     - uses: actions/cache@v4
       if: inputs.cache == 'true'

